### PR TITLE
Update Zendesk.java

### PR DESCRIPTION
**Pull request by mamsterla**
Fixed the template for URI to correctly accept passed in token

### CODE REVIEW

**Comment by mamsterla** [comment]
This simple change should allow repeated uploads using the same token

File: src/main/java/org/zendesk/client/v2/Zendesk.java [hunk 1]
```diff
@@ -428,7 +428,7 @@ public void deleteTicketField(long id) {
     }
 
     public Attachment.Upload createUpload(String token, String fileName, String contentType, byte[] content) {
-        TemplateUri uri = tmpl("/uploads.json{?filename}{?token}").set("filename", fileName);
+        TemplateUri uri = tmpl("/uploads.json{?filename}{&token}").set("filename", fileName);
```
Comment (stephenc) [inline src/main/java/org/zendesk/client/v2/Zendesk.java]: probably should be `{?filename,token}` instead

Comment (ydubreuil) [inline src/main/java/org/zendesk/client/v2/Zendesk.java]: I confirm, see here: https://github.com/cloudbees/zendesk-java-client/blob/master/src/main/java/org/zendesk/client/v2/Zendesk.java#L308

**Comment by stephenc** [comment]
I have made the changes in #123. Thank you for your contribution.

**Pull request closed without merging**